extensions/sql-migration/package.json

```diff
@@ -13,7 +13,8 @@
   },
   "activationEvents": [
     "onCommand:sqlmigration.start",
-    "onCommand:sqlmigration.testDialog"
+    "onCommand:sqlmigration.testDialog",
```

Comment:
test dialog? Should this actually be checked in? Typically this kind of thing should just be left as a local change for testing.

Reply:
This was added by Amir to open the migration wizard without any data. Will remove before final release.

Reply:
It would be good to remove it now (in a separate PR) so we don't forget - do you even need to use it anymore?

Reply:
This feature is used by @cssuh, to work on his UI. I can ask if he still wants it here